rosa_hcp/rosa-hcp-egress-lockdown-install.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-egress-lockdown-install"]
+= Creating a {product-title} cluster with egress lockdown
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-egress-lockdown-install
+toc::[]
+
+You can create a fully operational cluster that does not require a public egress by configuring a virtual private cloud (VPC) and using the `--properties zero_egress:true` flag when creating your cluster.
+
+.Prequisites
+
+* You have an AWS account with sufficient permissions to create VPCs, subnets, and other required infrastructure.
+* You have installed the Terraform v1.4.0+ CLI.
+* You have installed the ROSA v1.2.45+ CLI.
+* You have installed and configured the AWS CLI with the necessary credentials.
+* You have installed the git CLI.
+
+[id="rosa-hcp-creating-vpc-disconnected-install"]
+== Creating a Virtual Private Cloud for your disconected {hcp-title} clusters
+
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} clusters. You can use one of the following methods to create a VPC:
+
+* Create a VPC by using a Terraform template
+* Manually create the VPC resources in the AWS console
+
+[NOTE]
+====
+The Terraform instructions are for testing and demonstration purposes. Your own installation requires some modifications to the VPC for your own use. You should also ensure that when you use this Terraform script it is in the same region that you intend to install your cluster. In these examples, use `us-east-2`.
+====
+
+[discrete]
+include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+[id="additional-resources_rosa-hcp-disconnected-install-vpc-terraform"]
+.Additional resources
+
+* See the link:https://github.com/openshift-cs/terraform-vpc-example[Terraform VPC] repository for a detailed list of all options available when customizing the VPC for your needs.
```

Comment:
This needs to refer to the zero-egress subfolder in this repo if they want to create a VPC without egress.